tasks/libs/common/datadog_api.py

```diff
@@ -46,4 +45,10 @@ def send_metrics(series):
     configuration = Configuration()
     with ApiClient(configuration) as api_client:
         api_instance = MetricsApi(api_client)
-        return api_instance.submit_metrics(body=MetricPayload(series=series))
+        response = api_instance.submit_metrics(body=MetricPayload(series=series))
+
+        if response["errors"]:
+            print(f"Error(s) while sending pipeline metrics to the Datadog backend: {response['errors']}")
```

Comment:
```suggestion
            print(f"Error(s) while sending pipeline metrics to the Datadog backend: {response['errors']}", file=sys.stderr)
```
you would need to `import sys` as well